example/stochastic_subgrid/stochastic_subgrid_package.cpp

```diff
@@ -377,7 +377,7 @@ TaskStatus CalculateFluxes(std::shared_ptr<MeshBlockData<Real>> &rc) {
   const int nx1 = pmb->cellbounds.ncellsi(IndexDomain::entire);
   const int nvar = advected.GetDim(4);
   size_t scratch_size_in_bytes = parthenon::ScratchPad2D<Real>::shmem_size(nvar, nx1);
-  parthenon::ParArray4D<Real> x1flux = rc->Get("advected").flux[X1DIR].Get<4>();
+  parthenon::ParArray4D<Real> x1flux = rc->Get("bnd_flux::advected").data.Get(0, 0, 0);
```

Comment:
Ok what's with the `Get(1, 0, 0)`? Direction 0, but what are the other two 0s for?

Reply:
I believe `ParArrayND::Get<D>` returns a `D` dimensional sub array where the first `N-D` indices of the original array are set to zero. `ParArrayND::Get(int... args)` returns and `N-sizeof...(args)` dimension array with the leading indices set to `args...`. Because the leading index is the topological element index (i.e. flux direction index), we need to change a little bit about how we pull out the subarray.

Reply:
Yeesh I should have known that... since I wrote `ParArrayND::Get`...